astropy/_version.py

```diff
@@ -0,0 +1,4 @@
+# coding: utf-8
```

Comment:
This should not be added

Reply:
Argh, good catch. I had that file from `master`.